contrib/test/ci/integration-main.yml

```diff
@@ -12,6 +12,11 @@
       include_tasks: "build/kata.yml"
       when: "build_kata | default(False) | bool"
 
```

Comment:
now that we've upated the cri-tools in the VM image we should undo this change so we continue to use the (new) cached version

Reply:
we do it as part of [setup](https://github.com/cri-o/cri-o/blob/main/contrib/test/ci/setup.yml) which runs as a periodic job that we then cache the images for. basically: we update the images daily instead of doing it each ci run. The upstream doesn't change enough to warrant rebuilding each PR and spending all that download/build time.

Reply:
@haircommander 
aha, I should've waited for a while. thanks!

Reply:
no this is fine to force a rebuild temporarily, no worries :slightly_smiling_face: